plugins/janus_streaming.c

```diff
@@ -5590,6 +5590,20 @@ static int janus_streaming_rtsp_connect_to_server(janus_streaming_mountpoint *mp
 			JANUS_LOG(LOG_VERB, "  -- SSRC (video): %"SCNu32"\n", ssrc);
 			source->video_ssrc = ssrc;
 		}
+		if(strcmp(vhost, "0.0.0.0") == 0) {
```

Comment:
I see a couple of problems here:

1. You're only handling video for some reason: the same should be done for audio too.
2. Your trigger for checking `source` is the c-line being `0.0.0.0`, but it should actually also be when no c-line is there at all (maybe unlikely, but better handle that). To do that, both `ahost` and `vhost` should be initialized after they're declared, e.g.., `ahost[0] = '\0';`, as this way, if parsing the SDP we can't find any c-line, then we can check if the host string is empty as another trigger.